internal/operatorcli/operator_cli.go

```diff
@@ -84,28 +92,38 @@ func RunEncrypt(c *cli.Context) error {
 
 func RunDecrypt(c *cli.Context) error {
 	passphrase := c.String(flag.PassphraseFlag.Name)
+	blsPassphrase := c.String(flag.BlsPassphraseFlag.Name)
+	ecdsaPassphrase := c.String(flag.EcdsaPassphraseFlag.Name)
+	aliasedEcdsaPassphrase := c.String(flag.AliasedEcdsaPassphraseFlag.Name)
+
 	if passphrase == "" {
-		return cli.Exit("passphrase is required", 1)
+		if blsPassphrase == "" || ecdsaPassphrase == "" || aliasedEcdsaPassphrase == "" {
+			return cli.Exit("either passphrase or bls/ecdsa/aliased ecdsa passphrase is required", 1)
+		}
+	} else {
+		blsPassphrase = passphrase
+		ecdsaPassphrase = passphrase
+		aliasedEcdsaPassphrase = passphrase
 	}
```

Comment:
same comment on checking first the new flags and later the backward compatibility